### PR TITLE
adds exports for ActiveModel and JSONAPI serializer to index

### LINKED ISSUE #465: Probably export JSONAPISerializer + ActiveModelSerializer as named exports from index.js
since we're doing it with every other class

### PR DESCRIPTION
**Pull request by makepanic**
fixes #465 

Do we need tests that check for existence of these named exports?

### CODE REVIEW

**Comment by samselikoff** [comment]
Thank you! Umm, that would probably be a good idea :)

**Comment by samselikoff** [comment]
https://github.com/samselikoff/ember-cli-mirage/issues/471

File: addon/index.js [hunk 1]
```diff
@@ -3,6 +3,8 @@ import Response from './response';
 import faker from './faker';
 import Model from './orm/model';
 import Serializer from './serializer';
+import ActiveModelSerializer from './serializers/active-model-serializer';
+import JSONAPISerializer from './serializers/json-api-serializer';
```
Comment (cs3b) [inline addon/index.js]: shouldn't it be `JsonApiSerializer` to keep consistency (alternative, we should change that also in other places?)

Comment (samselikoff) [inline addon/index.js]: Let's stick with `JSONAPISerializer` to match Ember Data, and if it's different anywhere else, let's change it.

**Pull request merged**